packages/cli/docs-markdown-utils/src/parseImagePaths.ts

```diff
@@ -35,6 +35,13 @@ export function parseImagePaths(
 
     const filepaths = new Set<AbsoluteFilePath>();
 
+    if (data.image) {
+        const resolvedPath = resolvePath(data.image, metadata);
+        if (resolvedPath != null) {
+            filepaths.add(resolvedPath);
+        }
+    }
```

Comment:
@RohinBhargava worth adding a test so that this functionality isnt dropped in the future?